README.md

```diff
@@ -13,7 +13,7 @@ For best immediate results you should *not* load your system before you load Sta
 * `:documents` A list of documents for which to generate documentation pages. This usually refers to things like `README.md` and such. Typically you would only have multiple of these if you have translated the documentation into different languages. In that case, Staple will create an index page for each language individually.
 * `:page-type` The class to use for pages. See the sections below.
 * `:template` The template file to use. If you want to customise what the documentation pages look like, you might want to change this to one of your own.
-* `:if-exists` What to do if the output file already exists.
+* `:if-exists` What to do if the output file already exists (see [open](http://clhs.lisp.se/Body/f_open.htm)).
```

Comment:
It is standard for file operations to have an if-exists argument that mirrors OPEN's. I don't think it's necessary to mention it explicitly, but I guess it can't hurt either.